lib/foreman_docker/engine.rb

```diff
@@ -58,32 +58,41 @@ class Engine < ::Rails::Engine
 
         security_block :containers do
           permission :view_containers,
-                     :containers          => [:index, :show],
-                     :'api/v2/containers' => [:index, :show, :logs]
-          permission :commit_containers, :containers => [:commit]
+                     { :containers          => [:index, :show],
+                       :'api/v2/containers' => [:index, :show, :logs] },
+                     :resource_type => 'Container'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.